include/spirv-tools/libspirv.h

```diff
@@ -343,8 +343,13 @@ typedef spv_context_t* spv_context;
 
 // Platform API
 
-// Creates a context object.
-spv_context spvContextCreate();
+// Certain target environments impose additional restrictions on SPIR-V, so it's
+// often necessary to specify which one applies.  SPV_ENV_UNIVERSAL means
+// environment-agnostic SPIR-V.
+enum spv_target_env { SPV_ENV_UNIVERSAL, SPV_ENV_VULKAN };
```

Comment:
How do we accomodate evolution of the SPIR-V spec or the Vulkan spec?
It seems to me there be versions attached.
E.g. SPV_ENV_UNIVSERAL_1_0 and SPV_ENV_VULKAN_1_0
If we don't add the version distinction now, then the meaning of these non-versioned names will be ambiguous in the future.